javascript/javascript1/week4/homework6/scripts.js

```diff
@@ -0,0 +1,133 @@
+let todos = []; // initializing an empty list of todo
+let loggedName = []; //initializing an empty list of names
+
+//function to capture the name from the command
+function getName(command){
+    const name = command.split(' ')[4]; // note to self, split is a string method, it takes a separtor argument, in this case it's a space, and returns an array populated with the separeted elements as strings, I am accessing the 5th element directly
+    return name
+}
+
+//function to capture a task, either to add or remove
+function getTask(command) {
+    if(command.startsWith('Add') || command.startsWith('add')){
```

Comment:
Good job handling case sensitivity.

However, there could be other ways you might want to handle it - perhaps you could consider using `.toUpperCase()` or `.toLowerCase()` to normalize the string before your `if (...)` statement 😀

Reply:
That is a better approach. Thank you!